cloudbuild.yaml

```diff
@@ -25,81 +28,299 @@ steps:
   - container
   - clusters
   - get-credentials
-  - "limani-integ"
+  - cluster-1
   - --region
-  - "us-central1"
+  - us-west1-a
   - --project
-  - "$PROJECT_ID"
+  - $PROJECT_ID
+  waitFor:
+  - '-'
 
-- id: Initialized
-  name: bash
+- id: Pull cloud-marketplace-tools deployer_envsubst Image
+  name: gcr.io/cloud-builders/docker
+  entrypoint: /bin/bash
+  args:
+  - -exc
+  - |
+    MARKETPLACE_TOOLS_TAG=sha_7c70bcd83a8c490a11f93abfc4ec7a5f39c61ab6
+
+    docker pull gcr.io/cloud-marketplace-tools/k8s/deployer_envsubst:$$MARKETPLACE_TOOLS_TAG
```

Comment:
I don't think we need to pull these images, as `docker build` should pull them automatically?